### PR TITLE
Adds new data fields for prize location and willingness to ship inter…

### LINKED ISSUE #232: International shipping options
Some people offer prizes that are physical items which need to be shipped. The software should have a way to capture the location a prize donor's will be shipping from, and whether the donor is willing to ship internationally.

### PR DESCRIPTION
**Pull request by jmacdotorg**
…nationally, and appropriate display for this data when warranted. Fixes #232.

### CODE REVIEW

**Comment by jmacdotorg** [comment]
@mjmusante Should this PR have additional work WRT the MariaDB container? (Now that all the Docker stuff is part of the master branch...)

File: IFComp/lib/IFComp/Schema/Result/Prize.pm [hunk 1]
```diff
@@ -147,6 +158,10 @@ __PACKAGE__->add_columns(
     },
     is_nullable => 0,
   },
+  "location",
+  { data_type => "char", is_nullable => 1, size => 64 },
+  "ships_internationally",
+  { data_type => "tinyint", is_nullable => 1 },
```
Comment (adherzog) [inline IFComp/lib/IFComp/Schema/Result/Prize.pm:164]: I find nullable boolean fields can easily become a source of confusion. I'd suggest making this non-nullable with a default of 0.

File: IFComp/root/src/_prize_list.tt [hunk 2]
```diff
@@ -41,6 +41,9 @@ USE Markdown;
             <li>
             <p>[% IF prize.url %]<a href="[% prize.url %]">[% END %]<strong>[% prize.name | markdown | remove('</?p>') %]</strong>[% IF prize.url %]</a>[% END %]
             [% IF prize.notes %]<br />[% prize.notes | markdown | remove('</?p>') %][% END %]
+            [% IF prize.location && !prize.ships_internationally %]
+                <br /><em>This prize ships only to addresses within [% prize.location %].</em>
```
Comment (adherzog) [inline IFComp/root/src/_prize_list.tt:45]: Some of the other fields (`prize.name`, `prize.notes`) are put through the `markdown` filter. Should `prize.location` too? If not, maybe it should be put through the `html` filter?

File: IFComp/lib/IFComp/Schema/Result/Prize.pm (hunk 1 above)
Comment (jmacdotorg) [reply IFComp/lib/IFComp/Schema/Result/Prize.pm:164]: Good idea. In this case I'll set the default to 1, though, given that "I'm willing to ship internationally" is the current effective default that we want to provide an alternative to.

File: IFComp/root/src/_prize_list.tt (hunk 2 above)
Comment (jmacdotorg) [reply IFComp/root/src/_prize_list.tt:45]: Yes, I suppose that everyone who wishes to limit shipments to Turks & Caicos will appreciate that.

**Pull request merged**